content/applications/hr/employees/new_employee.rst

```diff
@@ -183,42 +195,48 @@ this section may be critical for the company's payroll department. In order to p
 payslips and ensure all deductions are accounted for, the employee's personal information should be
 entered.
 
-Here, the employee's :guilabel:`Private Contact`, :guilabel:`Marital Status`, :guilabel:`Emergency
-Contact`, :guilabel:`Education`, :guilabel:`Citizenship`, :guilabel:`Dependant`, and :guilabel:`Work
-Permit` information is entered. Fields are entered either using a drop-down menu, clicking a check
-box, or typing in the information.
+Here, the employee's :guilabel:`Private Contact`, :guilabel:`Education`, :guilabel:`Work Permit`,
+:guilabel:`Family Status`, :guilabel:`Emergency Contact`, and :guilabel:`Citizenship` information is
+entered. Fields are entered either using a drop-down menu, clicking a check box, or typing in the
+information.
 
-- :guilabel:`Private Contact`: Enter the :guilabel:`Address` for the employee. The selection can be
-  made with the drop-down menu. If the information is not available, type in the name for the new
-  address. To edit the new address, click the :guilabel:`External Link` button to open the address
-  form. On the address form, enter the necessary details, then click :guilabel:`Save`. Some other
+- :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The
```

Comment:
This is a very long paragraph/bullet point. Consider breaking it up into multiple sub-bullet points beneath the Private Contact bullet point?